recipes/libs/fontconfig.yaml

```diff
@@ -0,0 +1,46 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+    PKG_VERSION: "2.13.92"
+
+depends:
+    - name: devel::gettext
+      use: [result, deps, tools]
```

Comment:
Do you really need the result and the transitive dependencies? That would mean it's still broken because if fontconfig links with a binary of gettext it will have the wrong architecture. Does `use: [tools]` suffice?